.github/workflows/test-users.yml

```diff
@@ -0,0 +1,332 @@
+# Test a selection of projects which depend on rustix.
+
+name: Test rustix's users
+
+on: workflow_dispatch
+
+jobs:
+  async-io:
+    name: async-io ported to rustix
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        build: [stable]
+        include:
+          - build: stable
+            os: ubuntu-latest
+            rust: stable
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - uses: ./.github/actions/install-rust
+      with:
+        toolchain: ${{ matrix.rust }}
+
+    - run: sed -i 's/^version = "\([[:digit:].]*\)-alpha.0"$/version = "\1"/' Cargo.toml
+    - run: git clone https://github.com/sunfishcode/async-io
+    - run: cd async-io && echo '[patch.crates-io]' >> Cargo.toml
+    - run: cd async-io && echo 'rustix = { path = ".." }' >> Cargo.toml
+    - run: cd async-io && cargo test
+
+  cap-std:
+    name: cap-std
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        build: [stable]
+        include:
+          - build: stable
+            os: ubuntu-latest
+            rust: stable
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - uses: ./.github/actions/install-rust
+      with:
+        toolchain: ${{ matrix.rust }}
+
+    - run: sed -i 's/^version = "\([[:digit:].]*\)-alpha.0"$/version = "\1"/' Cargo.toml
```

Comment:
Related to this, personally I find the prior `cargo release` default of bumping to an alpha version causes much more pain than problems it solves.  I did this exact thing when I was trying to test a rustix patch locally too.

This is https://github.com/crate-ci/cargo-release/issues/82

Reply:
Makes sense; thanks for that link. It's counter-intuitive to me, but it seems like the way the tooling wants us to go here, so I'll stop using --dev-version.